### DISCUSSION TITLE
V3 Why Section Blade component get collapsed when I click on button for headerEnd slot?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#9509] [Help]
https://filamentphp.com/docs/3.x/support/blade-components/section#adding-content-to-the-end-of-the-header
https://filamentphp.com/docs/3.x/support/blade-components/section#making-a-section-collapsible
https://filamentphp.com/docs/3.x/support/blade-components/button

**my-page.blade.php**

```
<x-filament-panels::page>
  <x-filament::section icon="bi-trophy" icon-color="primary" collapsible>
    <x-slot name="heading">
      {{ strtoupper(__('What is Lorem Ipsum?')) }}
    </x-slot>

    <x-slot name="description">
      {{ ucfirst(__('Lorem Ipsum is simply dummy text of the printing and typesetting industry.')) }}
    </x-slot>

    <x-slot name="headerEnd">
      <x-filament::button :href="route('filament.admin.pages.about')" tag="a" icon="bi-eye">
        {{ __('View more') }}
      </x-filament::button>
    </x-slot>
  </x-filament::section>
</x-filament-panels::page>
```

https://github.com/filamentphp/filament/assets/121255432/1cb935d7-daf7-4da3-a49d-151b833b140f

**Comment by sinnbeck** [comment]
Try disabling spa mode

**Reply by devhoussam1998** [reply]
even I disable the spa mode still get same result

**Reply by sinnbeck** [reply]
So reloading the page does not reset it?

I assume the errors in the console are gone at least ?

**Reply by devhoussam1998** [reply]
yeah it's gone

**Reply by devhoussam1998** [reply]
but still have the button issue when section collapse
